model_compression_toolkit/core/pytorch/back2framework/pytorch_model_builder.py

```diff
@@ -84,6 +84,8 @@ def _run_operation(n: BaseNode,
     functional_kwargs = n.op_call_kwargs if isinstance(n, FunctionalNode) else {}
     if isinstance(n, FunctionalNode) and n.inputs_as_list:
         out_tensors_of_n = op_func(input_tensors, *op_call_args, **functional_kwargs)
+    elif n.type == BufferHolder:
+        out_tensors_of_n = n.weights['constant']
```

Comment:
Why do you need this? can't you implement it similar to ConstantHolder? add a "forward" method to BufferHolder